src/creatures/players/account/account.cpp

```diff
@@ -333,9 +333,7 @@ namespace account {
 			query << "`email` = " << db_->escapeString(accountIdentifier_) << " , ";
 		}
 
-		query << "`type` = " << account_type_ << " , "
```

Comment:
This was causing a weird issue. I'm also not entirely convinced we need to save the type and password every time we save the account?

Reply:
Indeed we shouldn't. As far as I can see, we only set password and type when we load from the DB. There is no application flow that manipulates those fields. I believe we are suffering here from having the same data model for reading and writing, but in fact we should never update password and account_type here.